### DISCUSSION TITLE
How can I expand bundle items deep in an API call?

### DISCUSSION THREAD

**Discussion by gsumo** [swellstores/community#197] [Help] [answered]
const { results, page_count, page, ...rest } = await swell.subscriptions.list({
  expand: ["orders"],
});

<img width="409" alt="Screenshot 2023-04-04 at 12 10 56 PM" src="https://user-images.githubusercontent.com/106490529/229896039-beaed81c-63cd-4c00-ba3c-ea1a1f0880d8.png">

product.bundle_items <-- I need the product names here

**Comment by ericingram** [comment, marked as answer]
`expand: product.bundle_items.product`
